dashboard/app/views/levels/editors/fields/_ai_tutor_fields.html.haml

```diff
@@ -0,0 +1,13 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#ai-tutor-container"}}
+  AI Tutor
+#ai-tutor-container.in.collapse
+
+%script{src: webpack_asset_path('js/levels/editors/fields/_ai_tutor_fields.js'),
+  data: {aitutoravailable: (@level.properties["ai_tutor_available"] || true).to_json, aitutorlevelspecificprompt: (@level.properties["ai_tutor_level_specific_prompt"] || '').to_json}}
+
+  -# data: {aicustomizations: (@level.properties["initial_ai_customizations"] || {}).to_json, levelname: @level.name, appname: @level.game&.app}
+    = f.label :ai_tutor_level_specific_prompt, 'Level-Specific Prompt'
+    %p Prompt used to specify the behavior of AI Tutor on a per level basis.
+    = f.text_area :ai_tutor_level_specific_prompt, placeholder: 'Write your level-specific prompt here. If left blank, this level will use the default system prompt.', rows: 4, class:"input-block-level"
```

Comment:
for my understanding: why do we need this and the react version?

Reply:
Good catch @molly-moen! That was the old version that I had commented out, I will remove.